CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ The version headers in this history reflect the versions of Apollo Server itself
 
 - _Nothing yet!_
 
+- `@apollo/federation`: `buildFederatedSchema` now accepts the same possible interface combination as `new ApolloServer()` for `typeDefs` and `resolvers` to make the migration from a single service into a federated service easier for teams with existing `typeDefs` arrays already built up. [PR #3188](https://github.com/apollographql/apollo-server/pull/3188)
```

Comment:
In order for "same possible interface combination" to be accurate, we'd also have to support strings and arrays of strings for `typeDefs`, since `ApolloServer`'s `typeDefs` currently allows:

```ts
const typeDefs: string | DocumentNode | DocumentNode[] | string[]
```

Of course, these are just turned into `DocumentNode`s (I think by `graphql-tools` still, actually), but it is an artifact of the past.  (I was recently surprised by this; we discovered this painfully via https://github.com/apollographql/apollo-server/pull/2754 https://github.com/apollographql/apollo-server/pull/2428 https://github.com/apollographql/apollo-server/pull/2762)

If you think this covers the main use cases, I think this is already a big win, but would suggest we change the `CHANGELOG.md` to read:

```suggestion
- `@apollo/federation`: `buildFederatedSchema`'s `typeDefs` parameter now accepts arrays of `DocumentNode`s (i.e. type definitions wrapped in `gql`) and `resolvers` to make the migration from a single service into a federated service easier for teams previously utilizing this pattern. [PR #3188](https://github.com/apollographql/apollo-server/pull/3188)
```